src/main/java/com/epam/controller/ToursController.java

```diff
@@ -136,4 +145,24 @@ public ModelAndView updateTour(@RequestParam String tourId, @RequestParam String
     public String getAddTour() {
         return "addtour";
     }
+
+    @PostMapping("/reserveTour")
```

Comment:
Have you tested it? I tried now, and get error during search the tour
How can I do it?

Reply:
Yes, it works fine, but because of some wrong column names you got these errors 
I fix it in another pr [https://github.com/Cisfinitum/FirstProject/pull/53]